docs/development/ADRs/cartesian/frontend-indexing-absolute-k.md

```diff
@@ -0,0 +1,74 @@
+# ⚠️ Absolute K indexing
+
+In the context of porting physics parametrizations, some field accesses were at absolute indices in the K-axis. We thus decided to expand the DLS be able to port these stencils. We accept the increase in DSL surface and test it as an [experimental feature](./general-experimental-features.md).
+
+## Context
+
+Porting physics parametrizations, we found two following two access patterns
+
+```fortran
+do L = 1, LM
```

Comment:
I know the L is describing layers here. Should we stick with IJK in the GT-world neverthenless for it to be easier to translate?